### DISCUSSION TITLE
Finding grbl responses

### DISCUSSION THREAD

**Discussion by jrlandau** [vlachoudis/bCNC#1798] [Q&A]
When in Control mode I enter a command in the little command window at the bottom of the screen, I do not see a response.  Even "$$" does not seem to get a response.  Am I missing a window somewhere? 

I see that in Terminal mode command responses are visible.  Are the command responses from commands issued in Control mode always shown if you switch to Terminal mode?

**Comment by JipeB** [comment]
You gave the answer. You have to switch to terminal mode to display the dialog between bCNC and grbl.
A command placed in MDI mode will return "ok" or an error message.
Of course, if you place an order like G0 X-200 Y-100, the result will be directly visible on the CNC.
But some commands that cause no movement will only be acknowledged with a terse "ok"

> I see that in Terminal mode command responses are visible. Are the command responses from commands issued in Control mode always shown if you switch to Terminal mode?

Yes, just scroll through the history of orders issued
